docs/debug.md

```diff
@@ -0,0 +1,61 @@
+# This document is intended for developers who want to setup a debugger for this project in VS Code.
```

Comment:
This heading is too long. This should be an opening statement imo. We should have a simpler heading like `## DEBUGGING`